molnsutil/molnsutil.py

```diff
@@ -411,6 +414,20 @@ def run_ensemble_map_and_aggregate(model_class, parameters, param_set_id, seed_b
             raise MolnsUtilException(notes)
     return {'result':res, 'param_set_id':param_set_id, 'num_sucessful':num_processed, 'num_failed':number_of_trajectories-num_processed}
 
+
```

Comment:
Looks like this function is only used for the commented out multiprocessing block below.  We should remove it if it is not used.